### PR TITLE
addition of custom commands folder and pages 

### PR DESCRIPTION
**Pull request by Ryheff24**
Added documentation on all the options within the custom commands menu.

Added a introduction page about variables

### CODE REVIEW

File: docs/custom-commands/_category_.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "label": "Custom Commands"
+}
```
Comment (aidenwallis) [inline docs/custom-commands/_category_.json:3]: newline

File: docs/custom-commands/Aliases.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+id: Command-Aliases
+sidebar_position: 2
+---
+
+# Aliases
+
+A command alias is used to add another command name without creating a new custom command.
+
+![alias](https://i.imgur.com/TxOqLqc.png)
```
Comment (aidenwallis) [inline docs/custom-commands/Aliases.md:10]: don't host these on an image hosting platform, put these inside the repo.

File: docs/custom-commands/Aliases.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Create an alias by typing in the box and using the Enter key on your keyboard or the Create button. Click the X on the alias to delete it.
+
+![alias](https://i.imgur.com/Egx1bog.gif)
```
Comment (aidenwallis) [inline docs/custom-commands/Aliases.md:14]: same as above^

File: docs/custom-commands/conditions.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+id: Command-Conditions
+sidebar_position: 5
+---
+
+# Conditions
+
+Conditions are options that affect how the command may work.
+
+![conditions](https://i.imgur.com/F7qrwoj.png)
```
Comment (aidenwallis) [inline docs/custom-commands/conditions.md:10]: put inside repo

File: docs/custom-commands/conditions.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+id: Command-Conditions
```
Comment (aidenwallis) [inline docs/custom-commands/conditions.md:2]: These shouldn't be separate pages, there should be a single page with clickable sections imo. Maybe a section called "Commands", which has a page called "Custom Commands"

File: docs/variables/variables-intro.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (aidenwallis) [inline docs/variables/variables-intro.md:1]: delete this page

File: docs/custom-commands/roles.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+id: command-roles
+sidebar_position: 4
+---
+
+# Roles
+
+Roles allow you to set who has access to each command. This can be set to everyone, non-subscriber, subscriber, VIP, Founder, Moderator, and Broadcaster but also can be used with custom roles. 
+
+:::info
+When no roles are selected, the command works for all users.
```
Comment (aidenwallis) [inline docs/custom-commands/roles.md:11]: Not sure if this is the best phrasing, maybe like ```If no roles are selected, it's triggerable by all users``` or something? I like the idea of saying "triggering commands" as a standard across all docs imo

File: docs/custom-commands/roles.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+---
+id: command-roles
+sidebar_position: 4
+---
+
+# Roles
+
+Roles allow you to set who has access to each command. This can be set to everyone, non-subscriber, subscriber, VIP, Founder, Moderator, and Broadcaster but also can be used with custom roles.
```
Comment (aidenwallis) [inline docs/custom-commands/roles.md:8]: Those are still just roles, should just specify it's the roles defined in *another section*

File: docs/custom-commands/reponsetype.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+Say will print the command response normally in chat.
+
+## Mention
+
+Mention will Mention the user in the command response.
+
+## Reply
+
+Reply uses Twitch's chat reply feature to send the command response.
```
Comment (aidenwallis) [inline docs/custom-commands/reponsetype.md:22]: Maybe link to the twitchsupport tweet or a help article showing what this is

File: docs/custom-commands/reponsetype.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+
+Fossabot has 4 response types, Say, Mention, Reply, and Whisper. 
+
+## Say
+
+Say will print the command response normally in chat.
+
+## Mention
+
+Mention will Mention the user in the command response.
```
Comment (aidenwallis) [inline docs/custom-commands/reponsetype.md:18]: All of these should have examples.

File: docs/custom-commands/reponsetype.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
+---
+id: Command-Response-Type
+sidebar_position: 3
+---
+
+# Response Type
+
+Response type is how Fossabot will reply when a command is triggered.
```
Comment (aidenwallis) [inline docs/custom-commands/reponsetype.md:8]: This seems like bad eng, maybe `Response type defines how Fossabot answers commands.`

**Review by aidenwallis — CHANGES_REQUESTED** [review]
- We should host images in repo to not rely on imgur being online, and everyone should have access to control/change them.
- Having a lot of pages puts strain on the user, it's much easier to scroll up and down a page to find what you need, than having to click back and forth and try to piece together info.

Thank you for the contribution!

File: docs/custom-commands/roles.md (hunk 8 above)
Comment (Ryheff24) [reply docs/custom-commands/roles.md]: When a roles page is made, I will update with a link on how roles work.

File: docs/custom-commands/reponsetype.md (hunk 10 above)
Comment (Ryheff24) [reply docs/custom-commands/reponsetype.md]: added images to each response type

**Comment by Ryheff24** [comment]
I merged the custom commands section to 1 page, added more images, and removed the page for variables intro.

**Review by aidenwallis — APPROVED** [review]
Thank you! lgtm

**Comment by aidenwallis** [comment]
It looks like your markdown is not consistent @Ryheff24 - please fix that then we can merge :) (you can run `yarn lint` locally to compare and ensure it's good to go) https://github.com/fossadev/fossabot-docs/runs/4799965034?check_suite_focus=true

**Comment by aidenwallis** [comment]
lgtm, merging this, ty!

**Pull request merged**